src/Component/Field.js

```diff
@@ -0,0 +1,275 @@
+import React,{useState} from 'react'
+
+function Field({fieldData}) {
+
+    const [text,setText] = useState('')
+    const [fieldValue,setFieldValue] = useState([]);
+    const [finalFormData, setFinalFormData] = useState([])
+
+    let allFields = [];
+    let element = ''
+
+    function handlerButtonTypeButton(){
+      console.log("Button Type Button Clicked !")
+    }
+
+    function handleButtonTypesubmit(){
+         console.log("Button Type Submit  clicked !")
+         setFinalFormData(fieldValue.map(field => field));
+    } 
+   
+    function handleButtonTypeReset() {
+        setText('')
+        setFinalFormData([]);
+    }
+
+
+   function fieldClickhandler(e) {
+        switch(e.target.type){
+          case 'button': 
+             handlerButtonTypeButton();
+             break;
+           case 'submit': 
+             handleButtonTypesubmit();
+             break;
+           case "reset":
+             handleButtonTypeReset();
+             break;
+        }
+   }
+
+    function handleOnchange(e,field) {
+        setText(e.target.value)
+    } 
+
+    function handleBulr(e,field){ 
+      console.log(e.target)
+      //apply validation here 
+        if(e.target.value==="") return
+        if(fieldValue.length===0) return setFieldValue([...fieldValue,{name:field.attributes.name,value:e.target.value}])
+        if(fieldValue.length>0){
+            if(fieldValue.find(field => field.name===e.target.value)===undefined){
```

Comment:
Why are we checking `field.name === e.target.value` ??

Reply:
- Here we store , each final input value in to array, 
- but before that will check , that the value for that field exist or not, 
- In array we have field.name,,,,and we check that with incoming value by e.targe.name.